### PR TITLE
Add active to awards

### PR DESCRIPTION
**Pull request by youcatastrophe**

### CODE REVIEW

**Comment by youcatastrophe** [comment]
After checking documentation and looking through file after file pertaining to Administrate, I'm having a terribly difficult time getting the "active" checkbox to default to being checked in Administrate. Do you have any suggestions? Thanks.

File: app/dashboards/award_dashboard.rb [hunk 1]
```diff
@@ -36,6 +38,7 @@ class AwardDashboard < Administrate::BaseDashboard
     :relevant_fields,
     :created_at,
     :updated_at,
+    :active,
```
Comment (sumeetjain) [inline app/dashboards/award_dashboard.rb:41]: Remove trailing whitespace

File: db/migrate/20160330200611_add_active_to_awards.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddActiveToAwards < ActiveRecord::Migration
```
Comment (sumeetjain) [inline db/migrate/20160330200611_add_active_to_awards.rb:1]: Has no descriptive comment (IrresponsibleModule)

File: app/dashboards/award_dashboard.rb [hunk 3]
```diff
@@ -13,6 +13,7 @@ class AwardDashboard < Administrate::BaseDashboard
     relevant_fields: Field::Number,
     created_at: Field::DateTime,
     updated_at: Field::DateTime,
+    active: Field::Boolean,
```
Comment (sumeetjain) [inline app/dashboards/award_dashboard.rb:16]: Avoid comma after the last item of a hash.

File: app/dashboards/award_dashboard.rb [hunk 4]
```diff
@@ -26,6 +27,7 @@ class AwardDashboard < Administrate::BaseDashboard
     :relevant_fields,
     :created_at,
     :updated_at,
+    :active,
```
Comment (sumeetjain) [inline app/dashboards/award_dashboard.rb:30]: Avoid comma after the last item of an array.

File: app/dashboards/award_dashboard.rb (hunk 1 above)
Comment (sumeetjain) [inline app/dashboards/award_dashboard.rb:41]: Avoid comma after the last item of an array.

Comment (sumeetjain) [inline app/dashboards/award_dashboard.rb:41]: Trailing whitespace detected.

File: app/dashboards/award_dashboard.rb [hunk 5]
```diff
@@ -44,6 +47,7 @@ class AwardDashboard < Administrate::BaseDashboard
   FORM_ATTRIBUTES = [
     :award_name,
     :relevant_fields,
+    :active,
```
Comment (sumeetjain) [inline app/dashboards/award_dashboard.rb]: Avoid comma after the last item of an array.

File: db/schema.rb [hunk 6]
```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160329194001) do
+ActiveRecord::Schema.define(version: 20160330200611) do
```
Comment (sumeetjain) [inline db/schema.rb:14]: Separate every 3 digits in the integer portion of a number with underscores(_).

**Pull request merged**